apps/web/src/hooks/useAddWalletModal.tsx

```diff
@@ -59,18 +67,43 @@ export default function useAddWalletModal() {
         content: (
           <AddWalletModal queryRef={query} onConnectWalletSuccess={onConnectWalletSuccess} />
         ),
-        headerText: 'Connect your wallet',
+        headerText: 'Sign in or create an account',
```

Comment:
let's actually keep "Connect Your Wallet" as the header here because they're already signed in so this header wouldn't make sense. I think the design is incorrect